.pre-commit-config.yaml

```diff
@@ -97,8 +98,6 @@ repos:
   hooks:
   - id: python-check-blanket-noqa
     stages: [manual]
-  - id: python-check-blanket-type-ignore
-    stages: [manual]
```

Comment:
In MyPy 0.940, this is now an option, and using MyPy, it not only tells you that you need an error code, it suggests the correct one! :)